data/json/items/fake.json

```diff
@@ -325,5 +325,23 @@
     "symbol": "@",
     "color": "brown",
     "flags": [ "ALLOWS_REMOTE_USE", "PSEUDO" ]
+  },
+  {
+    "id": "pseudo_archery_target_box",
```

Comment:
There could also be one of these for the ``"f_target"`` furniture

Reply:
I'm pretty sure that pseudo item is for the target furniture? at least, the recipe won't work without it installed as furniture, and will work with it installed